controller/workitem_blackbox_test.go

```diff
@@ -3392,3 +3381,25 @@ func (s *WorkItem2Suite) TestCreateAndUpdateWorkItemForEveryWIT() {
 		})
 	}
 }
+
+func (s *WorkItem2Suite) TestDeleteWorkitem() {
+	s.T().Run("ok", func(t *testing.T) {
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.WorkItems(1))
```

Comment:
Please use `t` instead of `s.T()` in sub-tests.

Reply:
@kwk addressed it as a part of this PR - 37e2d35